pom.xml

```diff
@@ -73,41 +70,47 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>aws-java-sdk</artifactId>
-            <version>1.11.264</version>
+            <version>1.11.687</version>
```

Comment:
I'm wondering if such "meta-plugin" should be in the Jenkins BOM. 
But nothing to do here.